ui/client/components/graph/NodeUtils.ts

```diff
@@ -1,32 +1,62 @@
+/* eslint-disable i18next/no-literal-string */
 import _ from "lodash"
 import fp from "lodash/fp"
 import * as ProcessDefinitionUtils from "../../common/ProcessDefinitionUtils"
 import ProcessUtils from "../../common/ProcessUtils.js"
 
+type NodeType = "Properties" | "_group" | string
+type Node = {
+  id: string,
+  type: NodeType,
+}
+type NodeId = Node["id"]
+
+type GroupNode = Node & {
+  nodes: Node[],
+  ids: NodeId[],
+}
+
+type Group = {
+  id: string,
+  nodes: NodeId[],
+}
+type GroupId = Group["id"]
+
+type Edge = {
+  from: NodeId,
+  to: NodeId,
+}
+
+type Process = {
+  nodes: Node[],
+  edges: Edge[],
+  properties: $TodoType,
+}
+
 class NodeUtils {
 
-  isNode = (obj) => {
+  isNode = (obj: {}): obj is Node => {
```

Comment:
I know it's refactor.. But can we do that kind of functions do as arrow function in one line? It looks much better.. :)